language-bindings/python/src/wamr/wasm_c_api/ffi.py

```diff
@@ -37,7 +37,7 @@
     current_file = Path(os.readlink(current_file))
```

Comment:
Please rename folder `wasm_c_api` to `wasm-c-api`, or CI will report error.

Reply:
Imports such as `import wamr.wasm_c_api.ffi as ffi` will return an invalid syntax!

Reply:
do I remove the `_` and `-`? The naming could be `wamr.wasmcapi` and `wamr.wamrapi`

Reply:
Sorry, the CI check rule may be too strict, wasmcapi and wamrapi are OK for me. But not sure whether @lum1n0us could help change the CI check to allow some special folder with `_`.

Reply:
I've already renamed it without the underscore, no problem!

Reply:
In previous commit I was following the Python's convention of using subfolder names with underscore. Since it was not passing the CI I decided to remove hyphens and underscored @lum1n0us